guides/release/routing/specifying-a-routes-model.md

```diff
@@ -224,3 +343,14 @@ export default Route.extend({
 ```
 
 And calling `modelFor` returned the result of the `model` hook.
+
+## Debugging models
+
+If you are having trouble getting a model's data to show up in the template, here are some tips:
+
+- return static, sample data as a test to see if the problem is really in the model hook, or elsewhere down the line
```

Comment:
Referring to previous comments, can we change the phrase  _"static, sample data"_ to hard-coded data or something that's equally common phrased?